### ISSUE TITLE
Glyph artefacts for 18 characters

### ISSUE THREAD

**Issue by tonton-pixel** [nomfoundation/font#2]
I wish to report some glyph issues in the current `NomNaTong V4.8` font, although I am not sure whether this is the right place, since this repository seems to be mainly about converting the PostScript font to various OTF/TTF formats...

I first noticed these glyph artefacts for Vietnam-sourced character glyphs in the Unicode code charts 14.0, but it appears they are originally present in the `Nom Na Tong` font itself.

Some of these artifacts are quite minor, but in a few cases, the continuity of strokes is definitely "broken", and some strokes are getting so thin they are becoming "invisible" or even missing entirely...

```
U+3A33  㨳
U+69E3  槣
U+265DE 𦗞
U+2813E 𨄾

U+6237  户
U+623E  戾
U+623F  房
U+6241  扁
U+6247  扇
U+6DDA  淚
U+7BC7  篇
U+22A4A 𢩊
U+22A5C 𢩜
U+2AB5A 𪭚

U+9632  防
U+963B  阻

U+21A38 𡨸

U+2AD66 𪵦
```
![U+3A33](https://user-images.githubusercontent.com/26766930/151716898-9caf94da-12bd-4262-a62a-024204130365.png)
![U+69E3](https://user-images.githubusercontent.com/26766930/151716899-622afd5e-e586-4c5c-a146-a6de6d723bdb.png)
![U+265DE](https://user-images.githubusercontent.com/26766930/151716900-6abca3a4-2874-4e79-9804-9d9785f2926b.png)
![U+2813E](https://user-images.githubusercontent.com/26766930/151716901-1ca2211d-55a6-4961-911a-2d98394f0912.png)
![U+6237](https://user-images.githubusercontent.com/26766930/151716902-10cf0c88-6217-46b9-8eaa-66f5a44ab485.png)
![U+623E](https://user-images.githubusercontent.com/26766930/151716903-2740c003-3f86-4ecd-acdf-284df0e09949.png)
![U+623F](https://user-images.githubusercontent.com/26766930/151716905-b7829a18-5fda-4121-9d6d-4f1c859b8e50.png)
![U+6241](https://user-images.githubusercontent.com/26766930/151716906-7ba7e530-f01c-44f9-a575-39720e5f5d5b.png)
![U+6247](https://user-images.githubusercontent.com/26766930/151716908-286a6598-e604-4076-9ac0-ce68961bd2b1.png)
![U+6DDA](https://user-images.githubusercontent.com/26766930/151716909-175050be-52b1-4921-82c5-a79be34e5ff3.png)
![U+7BC7](https://user-images.githubusercontent.com/26766930/151716911-3e97ffbb-fe42-42e9-9922-fdbd7cd9a26e.png)
![U+22A4A](https://user-images.githubusercontent.com/26766930/151716912-dd1f1c02-c0cd-4a0f-837c-de58248908da.png)
![U+22A5C](https://user-images.githubusercontent.com/26766930/151716913-f979c0f0-3a27-43d1-8231-84ff7f2d7ed3.png)
![U+2AB5A](https://user-images.githubusercontent.com/26766930/151716914-038b9329-ae65-43d0-b3da-9ecf84cef884.png)
![U+9632](https://user-images.githubusercontent.com/26766930/151716915-2a418416-b455-413b-8007-b0876c77553a.png)
![U+963B](https://user-images.githubusercontent.com/26766930/151716917-9402b56f-b9ae-476f-950d-46c412566926.png)
![U+21A38](https://user-images.githubusercontent.com/26766930/151716918-581f37de-d5c4-49c3-94d2-4df72550385c.png)
![U+2AD66](https://user-images.githubusercontent.com/26766930/151716920-b46d012e-4c48-4707-b5c3-9749cd473bd4.png)

**Comment by nomfoundation** [comment]
Hi Michael,

This is the right place to report such issues and we appreciate your doing so. We'll try to get these fixed in an upcoming version of the font. Not sure about the timing for the next version of the Unicode charts.

Thanks,

Lee

**Comment by nomfoundation** [comment]
Fixed in 5.0 @tonton-pixel, please verify in 5.0 or later build and close.
